components/script/dom/webidls/EventTarget.webidl

```diff
@@ -5,7 +5,7 @@
  * https://dom.spec.whatwg.org/#interface-eventtarget
  */
 
-[Abstract, Exposed=(Window,Worker,Worklet)]
+[Constructor, Exposed=(Window,Worker,Worklet)]
```

Comment:
So I'm not sure whether this is the correct change. Looking through the python code, codegen seems to only care about the `[Abstract]` extended attribute here:
https://github.com/servo/servo/blob/6361aed46d5246037d636669dc64664971e94279/components/script/dom/bindings/codegen/CodegenRust.py#L2012-L2016
... which affects what DOMClass the WebIDL interface derives from. Removing the `[Abstract]` attribute would result in `EventTarget` interface inheriting from `EventTargetTypeId::EventTarget`, which looks incorrect.

Reply:
Without removing [Abstract] interface EventTargetBinding::Wrap function doesn't exists.
Also one of the newly passed tests states:
> EventTarget must be primary interface of new EventTarget()
Which somewhat corresponds to EventTarget inheriting from EventTargetTypeId::EventTarget.
I base my assumption on Event and EventSource interfaces which seem similar.